### PR TITLE
Merge remote-tracking branch 'apache/trunk' into confluentinc/master

### PR DESCRIPTION
**Pull request by omkreddy**
```
Conflicts:
- build.gradle
- gradle/dependencies.gradle
- gradle/wrapper/gradle-wrapper.properties
- gradlew
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -143,7 +95,9 @@ ext {
       "--add-opens=java.security.jgss/sun.security.krb5=ALL-UNNAMED"
     )
 
-  userMaxForks = project.hasProperty('maxParallelForks') ? maxParallelForks.toInteger() : null
+  maxTestForks = project.hasProperty('maxParallelForks') ? maxParallelForks.toInteger() : Runtime.runtime.availableProcessors()
+  maxScalacThreads = project.hasProperty('maxScalacThreads') ? maxScalacParallelism.toInteger() :
+      Math.min(Runtime.runtime.availableProcessors(), 8)
```
Comment (ijuma) [inline build.gradle:100]: You probably need the mavenUrl, mavenPassword, mavenUsername changes to workaround KC-1802. I shared an internal link privately with you.

**Review by ijuma — COMMENTED** [review]
Thanks, I reviewed the files with conflicts and have one comment below.

File: build.gradle (hunk 1 above)
Comment (omkreddy) [reply build.gradle:100]: Thanks for the review.  I pushed a new commit to update maven properties.

**Review by ijuma — APPROVED** [review]
LGTM, thanks.

**Pull request merged**